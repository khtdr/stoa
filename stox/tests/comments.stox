#!/usr/bin/env stox

print "checkpoint 1";
// that's a shebang comment up there, and this is a double slash one.

print "checkpoint 2";
/* you can have c-style comments as well */

print "checkpoint 3";
/**
 * those can /*
         even be nested ?! */
         print "checkpoint NO";
 */


print "checkpoint 4";
// but it does complain...

print "checkpoint 5";
/* if you forget to close it.

print "checkpoint NO";
