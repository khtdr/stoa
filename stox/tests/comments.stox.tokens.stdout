PRINT[3,1]
STRING[3,7]
SEMICOLON[3,21]
PRINT[6,1]
STRING[6,7]
SEMICOLON[6,21]
PRINT[9,1]
STRING[9,7]
SEMICOLON[9,21]
PRINT[17,1]
STRING[17,7]
SEMICOLON[17,21]
PRINT[20,1]
STRING[20,7]
SEMICOLON[20,21]
